pkg/operator/cassandra/constants/constants.go

```diff
@@ -0,0 +1,33 @@
+package constants
+
+// Labels
+const (
+	ClusterNameLabel    = "cassandra.rook.io/cluster"
+	DatacenterNameLabel = "cassandra.rook.io/datacenter"
+	RackNameLabel       = "cassandra.rook.io/rack"
+	SeedLabel           = "cassandra.rook.io/seed"
+)
+
+// Environment Variable Names
+const (
+	EnvVarPodIP        = "POD_IP"
+	EnvVarPodName      = "POD_NAME"
```

Comment:
pod name and namespace are already defined in https://github.com/rook/rook/blob/master/pkg/operator/k8sutil/k8sutil.go#L45, could you reuse those?

Reply:
Done !